### PR TITLE
chore: Add Viewport tags to all skeletons to fix mobile look

### PR DESCRIPTION
**Pull request by fredmaiaarantes**
To optimize the mobile viewing experience, it's recommended to add viewport tags to our apps.

This simple fix, adds them to all our skeletons and can ensure that the content on the website appears properly on different mobile devices, regardless of their screen size or orientation. After this is released, we can also remove [one step](https://react-tutorial.meteor.com/simple-todos/01-creating-app.html#1-6-Mobile-look) from our tutorials, making it easier to create new Meteor apps.

### CODE REVIEW

**Review by StorytellerCZ — APPROVED** [review]

**Review by zodern — CHANGES_REQUESTED** [review]
Having viewport tags in new apps will be nice. I'm surprised Meteor didn't already do this.

File: tools/static-assets/skel-apollo/client/main.html [hunk 1]
```diff
@@ -1,5 +1,13 @@
 <head>
   <title>Apollo Skeleton</title>
+  <meta charset="utf-8"/>
```
[the remark this answers is not in the document]
Comment (fredmaiaarantes) [reply tools/static-assets/skel-apollo/client/main.html]: Thanks, I removed it.

File: tools/static-assets/skel-apollo/client/main.html [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+  <meta http-equiv="x-ua-compatible" content="ie=edge"/>
+  <meta
+      name="viewport"
+      content="width=device-width, height=device-height, viewport-fit=cover, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
```
[the remark this answers is not in the document]
Comment (fredmaiaarantes) [reply tools/static-assets/skel-apollo/client/main.html]: Tks, I'm switching to having simpler tags as recommended by W3Schools.
https://www.w3schools.com/css/css_rwd_viewport.asp

File: tools/static-assets/skel-apollo/client/main.html [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+  />
+  <meta name="mobile-web-app-capable" content="yes"/>
+  <meta name="apple-mobile-web-app-capable" content="yes"/>
```
[the remark this answers is not in the document]
Comment (fredmaiaarantes) [reply tools/static-assets/skel-apollo/client/main.html]: Tks, I'm switching to having simpler tags as recommended by W3Schools.
https://www.w3schools.com/css/css_rwd_viewport.asp

**Pull request closed without merging**